test/ref/resize.cpp

```diff
@@ -444,7 +444,7 @@ TEST_CASE(resize_fail_test_5)
                            {"coordinate_transformation_mode", "pytorch_half_pixel"}}),
         {a0});
     p.compile(migraphx::make_target("ref"));
-    EXPECT(test::throws([&] { p.eval({}).back(); }));
+    EXPECT(test::throws([&] { std::ignore = p.eval({}).back(); }));
```

Comment:
Actually I think we can remove the call to `.back()` for this test. Its only checking that `eval` throws, it doesnt make sense to call `.back()` here.